site_gen.sh

```diff
@@ -17,4 +17,10 @@ else
   exit 0
 fi
 
-echo "  - "$CLUSTER_NAME >> ./sites/kustomization.yaml
\ No newline at end of file
+kustom="./manifests/sites/kustomization.yaml"
+# Ensure a newline exists on last line in kustomization
+last_byte=$( tail -c 1 ${kustom} | xxd -ps )
```

Comment:
why do you need this bit?

Reply:
The version of kustomization.yaml that is currently checked in does not have a trailing newline. Without this check for the newline (and adding it if missing) the resulting kustomization looks like this:
```apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

resources:
  - cnf-cu-du-site
  - sample-site-cu
  - sample-site-du  - test_site
```